src/datadec/ingest/types.py

```
@@ -0,0 +1,26 @@
+    def extension(self) -> str:
+        match self:
+            case (
+                TaskArtifactType.PREDICTIONS
+                | TaskArtifactType.RECORDED_INPUTS
+                | TaskArtifactType.REQUESTS
+            ):
+                return "jsonl"
+            case TaskArtifactType.METRICS:
+                return "json"
+            case _:
+                raise ValueError(f"Invalid task artifact type: {self}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)